test/e2e/grpc_test.go

```diff
@@ -35,14 +35,31 @@ import (
 
 type grpcTest func(*testing.T, test.ResourceNames, *test.Clients, string, string)
 
-func unaryTest(t *testing.T, names test.ResourceNames, clients *test.Clients, host, domain string) {
-	t.Helper()
-	t.Logf("Connecting to grpc-ping using host %q and authority %q", host, domain)
-	conn, err := grpc.Dial(
+func dial(host, domain string) (*grpc.ClientConn, error) {
+	if host != domain {
```

Comment:
when does this happen?

Reply:
This is when using ResolvableDomain = false, see line 186